glusterd2/commands/volumes/volume-create.go

```diff
@@ -100,7 +101,8 @@ func registerVolCreateStepFuncs() {
 func volumeCreateHandler(w http.ResponseWriter, r *http.Request) {
 
 	ctx := r.Context()
-	ctx, span := trace.StartSpan(ctx, "/volumeCreateHandler")
+	fnName := utils.GetFuncName()
+	ctx, span := trace.StartSpan(ctx, "/"+fnName)
```

Comment:
You really don't need to get the function name programatically here. You'd need the helper utility if `StartSpan` could be called dynamically at runtime, and you wouldn't know the callers before hand. But here you know the caller,  so you could just hard code the function name string whenever you call it. Just a simple `ctx, span := trace.StartSpan(ctx, "/volumeCreateHandler")` is good here. 
The same also applies to the other `StartSpan` calls.

Reply:
I will revert it here and hard code it in the other places.